### ISSUE TITLE
VoTT crash when using a big dataset

### ISSUE THREAD

**Issue by Miglas13** [Microsoft/VoTT#195]
I have a big dataset (approximately 8000 images) and when I am exporting tags VoTT crash because of Insufficient memory. Still it writes some files to HDD but not all of them. I am using the most recent application.

**Comment by aribornstein** [comment]
Yes this seems to be a known issue I am working on a better solution as well as a cloud based system to enable distributed tagging. Can you send me the error messages from the debug console of VoTT?

**Comment by Miglas13** [comment]
Yes. it says: Failed to load resource: net::ERR_INSUFFICIENT_RESOURCES in some images (I guess the ones he doesn't write the tags). I think it is loading all images to memory and this is what causes this problem. I have 32Gb of RAM and when I make export tags it fills all the memory of my PC. After 5 min running the app freezes all blank and I have to refresh to use again.

**Comment by wbreza** [comment]
@Miglas13 - We've just released v2.0.0 which should address your issue.  Please try it out.
